doc/modules/mod_auth_token.md

```diff
@@ -15,44 +15,34 @@ Generation of keys necessary to sign binary tokens is delegated to module `mod_k
 
 ### Options
 
-#### Validity periods
+#### `modules.mod_auth_token.validity_period`
+* **Syntax:** Array of TOML tables with the following keys: `token`, `value`, `unit` and following values: {token = `values: "access", "refresh", "provision"`, value = `non-negative integer`, unit = `values: "days", "hours", "minutes", "seconds"`}.
+* **Default:** `[{token = "access", value = 1, unit = "hours"}, {token = "refresh", value = 25, unit = "days"}]`
+* **Example:** `[{token = "access", value = 13, unit = "minutes"}, {token = "refresh", value = 13, unit = "days"}]`
 
 Validity periods of access and refresh tokens can be defined independently.
-
-Allowed units are:
-
-* days
-* hours
-* minutes
-* seconds
-
-The default values for tokens are:
-
-* 1 hour for an access token
-* 25 days for a refresh token
-
-Example configuration from `mongooseim.cfg`, inside `modules` section:
-
-```erlang
-{modules, [
-    {mod_auth_token, [{{validity_period, access}, {13, minutes}},
-                      {{validity_period, refresh}, {13, days}}]
-]}.
-```
-
 Validity period configuration for provision tokens happens outside the module since the server does not generate provision tokens - it only validates them.
 
 #### Required keys
 
 Keys are used for signing binary tokens using an HMAC with SHA-2 family function SHA-384.
 Therefore, `mod_auth_token` requires `mod_keystore` to provide some predefined keys.
```

Comment:
Should we not just reference `mod_keystore` here?

Reply:
You mean something like `please refer to.... section` instead of the whole description?

Reply:
That was my intention, since now it literally describes another module's configuration